src/analysis/normed/field/unit_ball.lean

```diff
@@ -91,6 +102,13 @@ instance [normed_division_ring 𝕜] : group (sphere (0 : 𝕜) 1) :=
 instance [normed_division_ring 𝕜] : has_distrib_neg (sphere (0 : 𝕜) 1) :=
 subtype.coe_injective.has_distrib_neg (coe : sphere (0 : 𝕜) 1 → 𝕜) (λ _, rfl) (λ _ _, rfl)
 
+@[simp, norm_cast]
+lemma coe_one_unit_sphere [normed_division_ring 𝕜] : ((1 : sphere (0 : 𝕜) 1) : 𝕜) = 1 := rfl
+
+@[simp, norm_cast]
+lemma coe_mul_unit_sphere [normed_division_ring 𝕜] (x y : sphere (0 : 𝕜) 1) :
```

Comment:
Can you add the lemma for `pow` too?

Reply:
@eric-wieser Could you please have another look?